src/index.js

```diff
@@ -45,4 +45,9 @@ module.exports = function ({
   };
 };
 
-Object.assign(module.exports, { addExecutionTag: ExecutionTags.addTag });
+Object.assign(module.exports, {
+  addExecutionTag: ExecutionTags.addTag,
+  info: LumigoLogger.info,
+  warn: LumigoLogger.warn,
+  error: LumigoLogger.error,
+});
```

Comment:
Any tests?

Reply:
I didnt get it yet, if `info` and `warning` do nothing. don't expose them

Reply:
I dont think error its clear enough 
```suggestion
Object.assign(module.exports, {
  reportError: 
});
```